torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -1211,6 +1216,14 @@ void Reducer::search_unused_parameters(
         "flag off. Note that this warning may be a false positive if your model "
         "has flow control causing later iterations to have unused parameters.");
   }
+  if (!static_graph_) {
```

Comment:
nit: if (!static_graph_ && ddp_graph_static_), so that the {} codes can be skipped if ddp_graph_static_ is false